front/app/components/IdeaButton/utils.ts

```diff
@@ -1,17 +1,13 @@
-import { MessageDescriptor } from 'react-intl';
-import { InputTerm, ParticipationMethod } from 'services/participationContexts';
-
+import { InputTerm } from 'services/participationContexts';
+import { Props } from '.';
 import { getInputTermMessage } from 'utils/i18n';
 import messages from './messages';
 
 export const getButtonMessage = (
-  participationMethod: ParticipationMethod,
-  buttonText: MessageDescriptor | undefined,
+  participationMethod: Props['participationMethod'],
```

Comment:
`buttonText` was not used.

`participationMethod` can only be `ideation` or `native_survey` if this component is rendered.

Reply:
Using `Props` here to keep it in sync with the component where this function is used.